src/chainstate/stacks/index/marf.rs

```diff
@@ -607,6 +600,9 @@ impl MARF {
         test_debug!("Set {}::{} = {}", BLOCK_HEIGHT_TO_HASH_MAPPING_KEY, height, next_block_hash.to_hex());
         test_debug!("Set {}::{} = {}", BLOCK_HASH_TO_HEIGHT_MAPPING_KEY, next_block_hash.to_hex(), height);
 
+        keys.push(OWN_BLOCK_HEIGHT_KEY.to_string());
```

Comment:
Can you do me a favor and add a `test_debug!()` like those above to show that we're inserting `OWN_BLOCK_HEIGHT_KEY` here?

Reply:
Sure

Reply:
Addressed in f2063f7ef72f3501806c53c3b6f2b12c43a00b93